### PR TITLE
feat: Update settings sections

### PR DESCRIPTION
**Pull request by zatteo**
To fit with the structure of the new settings app.

```
### ✨ Features

* Add and move links in dropdown settings
```

### CODE REVIEW

File: src/components/Settings/SettingsContent.jsx [hunk 1]
```diff
@@ -14,38 +14,93 @@ const SettingsContent = ({
   onClaudy,
   isDrawer = false,
```
Comment (zatteo) [inline src/components/Settings/SettingsContent.jsx:15]: What is the meaning of the isDrawer ? Is there a mobile responsive version ?

Comment (Crash--) [reply src/components/Settings/SettingsContent.jsx:15]: Yep. But I don't know if this api is still used :/

File: src/components/Settings/SettingsContent.jsx [hunk 2]
```diff
@@ -14,38 +14,93 @@ const SettingsContent = ({
...
+                </a>
+              </li>
+            </>
+          )}
+        </ul>
+        <ul className="coz-nav-group">
+          <li className="coz-nav-settings-item">
+            <a
+              role="menuitem"
+              href={`${settingsAppURL}#/permissions/slug`}
```
Comment (Crash--) [inline src/components/Settings/SettingsContent.jsx:72]: can you hide this link behind this flag? https://github.com/cozy/cozy-settings/blob/e7ee451bc5feb4540f47a00e0683f247658338a4/src/components/Sidebar.tsx#L80

File: src/components/Settings/SettingsContent.jsx (hunk 1 above)
Comment (zatteo) [reply src/components/Settings/SettingsContent.jsx:16]: It seems to be used at the root of the Bar for the moment

**Review by Crash-- — APPROVED** [review]

**Pull request merged**